examples-gallery/plot_sliding_block.py

```diff
@@ -0,0 +1,306 @@
+
+# %%
+""""
+sliding a block on a road
+=========================
+A block, modeled as a particle is sliding on a road to cross a
+hill. The block is subject to gravity and speed dependent
+friction. 
+Gravity points in the negative Y direction.
+A force tangential to the road is applied to the block.
+Three objective functions to me minimized may be selected:
```

Comment:
Broke RST formatting:
![image](https://github.com/csu-hmc/opty/assets/276007/22a1ffb9-4a10-438f-a418-48ecd23f9856)

Reply:
There was a whitespace after the "and" in the second line, which I failed to remove.
(I got VSC to show the trailing white spaces, but not yet to remove them automatically, will need my son's help for this)